samples/sensor/fxos8700/sample.yaml

```diff
@@ -6,3 +6,13 @@ tests:
     platform_whitelist: frdm_k64f hexiwear_k64 warp7_m4
     tags: samples sensor
     depends_on: i2c
+  test-accel:
+    harness: sensor
+    platform_whitelist: frdm_kw41z reel_board
+    tags: samples sensor
+    depends_on: i2c gpio
+    extra_configs:
+      - CONFIG_FXOS8700_MODE_ACCEL=y
+      - CONFIG_FXOS8700_MOTION=y
+      - CONFIG_FXOS8700_MOTION_INT1=y
+      - CONFIG_FXOS8700_PM_LOW_POWER=y
```

Comment:
Can we move these configs to a new `proj_motion.conf`?

Reply:
Done, but I have turned on motion and pulse detection in sample.yaml